src/core/attribution/markovChain.js

```diff
@@ -91,28 +91,60 @@ export function uniformDistribution(n: number): Distribution {
   return new Float64Array(n).fill(1 / n);
 }
 
+/**
+ * Distribution that is 1 at the indicator value and 0 elsewhere.
+ */
+export function indicatorDistribution(
```

Comment:
We should have a test for this function, too.
(Should be simple. :slightly_smiling_face:)

Reply:
Done! Thanks. :)